qb_upload_limit_per_day.py

```diff
@@ -3,14 +3,14 @@
 import time
 import datetime
 import schedule
-
+from os.path import exists
 
 # Configuration
 UPLOAD_LIMIT = 50  # Upload limit in GB (per day)
-QB_URL = "http://localhost:8080"  # qBittorrent Web UI URL
+QB_URL = "http://localhost:8080"
```

Comment:
```suggestion
QB_URL = "http://localhost:8080"  # qBittorrent Web UI URL
```
Those comments are there to describe users what it is.

Reply:
Sorry, it was as my url is different and I did a bunch of copy/paste and missed the comment